hips/draw/simple.py

```diff
@@ -83,112 +96,67 @@ def dst(self) -> np.ndarray:
              [0, 0]],
         )
 
-    @property
-    def projection(self) -> ProjectiveTransform:
+    def projection(self, tile: HipsTile) -> ProjectiveTransform:
         """Estimate projective transformation on a HiPS tile"""
-        corners = self.tile.meta.skycoord_corners.to_pixel(self.geometry.wcs)
+        corners = tile.meta.skycoord_corners.to_pixel(self.geometry.wcs)
         src = np.array(corners).T.reshape((4, 2))
         dst = self.dst
         pt = ProjectiveTransform()
         pt.estimate(src, dst)
         return pt
 
-    def warp_image(self) -> np.ndarray:
+    def fetch_tiles(self) -> 'HipsTile':
+        """Fetch HiPS tiles from a remote URL"""
```

Comment:
From the docs it's not clear that this is a generator.
If the Type annotation doesn't work properly, then at least mention that this is a generator that returns `HipsTile` objects in the docstring.